app/src/CreateRoomCard.js

```diff
@@ -30,8 +31,7 @@ class CreateRoomCard extends React.Component {
           <Formik
             initialValues={{ 'userName': '', 'roomName': '', 'password': '' }}
             validate={values => {
-              const errors = {};
-              return errors;
+              return {};
```

Comment:
If we're just returning an object without doing any additional processing, you could directly do
```suggestion
values => {}
```